src/waveform/renderers/allshader/waveformrenderersignalbase.h

```diff
@@ -15,6 +16,13 @@ class WaveformRendererSignalBase;
 class allshader::WaveformRendererSignalBase : public ::WaveformRendererSignalBase,
                                               public allshader::WaveformRendererAbstract {
   public:
+    enum Option {
+        None = 0b0,
+        SplitStereoSignal = 0b1,
+        HighDetails = 0b10,
```

Comment:
In American English, this should just be "High Detail", not plural Details.  (High modifies "the level of detail" as an adjective. It is ok to say "More detail" and also "more details", but with "high", only the singular form sounds right.)